### ISSUE TITLE
Export test items to CSV

### ISSUE THREAD

**Issue by aliaksandrbasau** [reportportal/reportportal#397]
Sometimes I need to get a list of filtered test items and it's very hard to do because the table that shows the test items is not really HTML table and useful shortcuts (like Ctrl+mouse_select in Firefox) don't allow to copy table cells.
It would be great to be able to export the table to some simple format like CSV so it can be manipulated.
Examples of pages when it should be available:
![image](https://user-images.githubusercontent.com/6681200/38957690-38fa305a-4329-11e8-92cb-a51b70db1c5a.png)
![image](https://user-images.githubusercontent.com/6681200/38957749-6305c0ee-4329-11e8-99ee-264c2b003f60.png)

**Comment by evjlobanova** [comment]
@aliaksandrbasau could you please clarify what reason you want to copy table for? Please provide use case and business goal?

**Comment by aliaksandrbasau** [comment]
@evjlobanova E.g. yesterday I needed to get a list of tests that had some particular failure and then to merge it in Excel with another list that had additional information about the test suites so I can see the pattern.
Sometimes I need to get a list of test suites with new failures so I can re-run only that portion of tests.

**Comment by evjlobanova** [comment]
@aliaksandrbasau so your main goal is to compare two test suites of launch and find a test with new failures, am I right?

**Comment by aliaksandrbasau** [comment]
@evjlobanova Not exactly. In that particular case I had to merge the list of test suites affected by some particular issue with the list that contains VM names where the suites were executed. After sorting by two columns (VM name and start time) I was able to determine that the issue affected only a few VMs and reproduced during particular time frame.
Usually I need to get the list of test items or failures for some unique reason. It happens approximately every two weeks and I forced to write Notepad++ script to extract needed information from the text copied from the RP page.
Export to CSV is rather standard feature for software that shows tables of records (e.g. Jira). And almost every software allows to export data to some format.

**Comment by DzmitryHumianiuk** [comment]
@aliaksandrbasau we have an export into Excel format (XLS)
it will represent all test items under launch.

you can use filter in excel to identify required set of test items.

**Comment by aliaksandrbasau** [comment]
@DzmitryHumianiuk Found it. It could work for me but it splits test items into pages which are separate tables and can't be filtered together. (And the export reliably fails for my largest launch with 70,000+ test items (6,000+ tests)).
